app/configurator/components/chart-annotations-selector.tsx

```diff
@@ -20,54 +21,65 @@ export const ChartAnnotationsSelector = ({
 }: {
   state: ConfiguratorStateDescribingChart;
 }) => {
+  const { activeField, meta } = state;
   const panelRef = useRef<HTMLDivElement>(null);
   useEffect(() => {
     if (panelRef && panelRef.current) {
       panelRef.current.focus();
     }
-  }, [state.activeField]);
+  }, [activeField]);
   const locale = useLocale();
 
-  const af = state.activeField === "title" ? "title" : "description";
+  const isInteractiveFilterField = useMemo(() => {
+    switch (activeField as InteractiveFilterType) {
+      case "dataFilters":
+      case "legend":
+      case "timeRange":
+      case "timeSlider":
+        return true;
+
+      default:
+        return false;
```

Comment:
Why could we have the default case ?

Reply:
I'll also need to probably improve this, the case here is that activeField could be a lot of things – areaLayer, segment, x, y, but also interactive filter and here I wanted to catch these cases in more clear way, for all other strings the default case if triggered. Do you have an idea how to handle this better?